src/shared/prosemirror-plugins/image-upload.ts

```diff
@@ -286,6 +310,41 @@ export class ImageUploader extends PluginInterfaceView<
         event.stopPropagation();
     }
 
+    getAcceptedFileTypesString(types: string[]): string {
+        if (types?.length === 0) {
+            return "";
+        }
+
+        let uploadCaptionString = types[0].replace(/image\//g, "");
+        if (types.length > 1) {
+            uploadCaptionString = types
+                .map((type, i) => {
+                    return i === types.length - 1
+                        ? type
+                        : i === types.length - 2
+                          ? `${type}, or`
```

Comment:
I wonder if we need this extra complication here. I would personally challenge the requirement and ask design if we can drop the or here and just use commas as separators. On top of that `or` is not the same in every language and should not be hardcoded.

Reply:
Maybe @kristinalustig can double check for us